src/java/org/apache/cassandra/tcm/EpochAwareDebounce.java

```diff
@@ -62,66 +45,46 @@ private EpochAwareDebounce()
      * calls to peers, retrieving a LogState which can be applied locally to produce the necessary {@code
      * ClusterMetadata}. The function takes a {@code Promise<LogState>} as input, with the expectation that this
      * specific instance will be used to provide blocking behaviour when making the rpc calls that fetch the {@code
```

Comment:
should remove this too: `The function takes a {@code Promise<LogState>} as input, with the expectation that this specific instance will be used to provide blocking behaviour when making the rpc calls that fetch the {@code
LogState}.`